cogs/sfu/courses.py

```diff
@@ -14,10 +14,11 @@ def __init__(self, bot):
         super().__init__(bot)
 
         # Add commands to the sfu group defined in the base class
-        self.sfuGroup.add_command(commands.Command(self.lookup, name="course"))
+        self.sfuGroup.add_command(self.course)
```

Comment:
I think a decorator like `sfuGroup.command()` would do the same thing?

Reply:
Cannot use a decorator with self, python complains.

Reply:
Cannot use sfuGroup decorate here as well because it is not defined in this file.